codesydney/taskstech
# Inventory list

## Rationale

Inventory List for all inventory items for each trader.

## Advice for Reviewers & Testing Notes

1. Please go Swagger and get Token with
username: trader1234@gmail.com
password: Password

2. Paste into src/actions/action.js  Line81  inside of {} , headers:{Authorization:`Bearer {Token should be get from Swagger} `}
3. go to url localhost:3000/inventory/list
4. You can check API request result by console in Devtools of your website.

In this branch, there is no login functions merged yet, so we need check manually....

## Screenshots:

- Add a screenshot or two of your changes, where applicable.
- 
![Screen Shot 2021-09-02 at 5 48 58 pm](https://user-images.githubusercontent.com/65881354/131805377-289e411a-7602-43b0-84b3-61139843cf73.png)


## Task Name
Create Page and functions for list all inventory items from DB.

- Issue Title


## Linting Checklist
